components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/model/SAMLSSOServiceProviderDO.java

```diff
@@ -590,4 +612,21 @@ public boolean isSamlECP() {
         return samlECP;
     }
 
+    /**
+     * Get IdP Entity ID alias.
+     *
+     * @return IdP Entity ID Alias
+     */
+    public String getIdpEntityIDAlias() {
+        return idpEntityIDAlias;
+    }
+
+    /**
+     * Set IdP Entity ID alias.
+     *
+     * @param idpEntityIDAlias
+     */
+    public void setIdpEntityIDAlias(String idpEntityIDAlias) {
+        this.idpEntityIDAlias = idpEntityIDAlias;
```

Comment:
Add a new line after the method signature.

Reply:
Addressed in https://github.com/wso2/carbon-identity-framework/pull/3100